.github/ISSUE_TEMPLATE/improvement-template-.md

```diff
@@ -1,12 +1,3 @@
----
-name: 'Improvement Template '
```

Comment:
Why are we changing templates in this pull request? If they need to be changed, in the future, I think it would be better to keep our pull requests contain a single fix, so I would split this PR, 1 for the bug fix, one for the template change

Reply:
I amnot changing the template. For some reasons, the template wasn't showing up in the template list because assign was empty. ok. that's a good point. will keep it in mind :)